UserControls/packages.config

```diff
@@ -1,10 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="HEC.MVVMFramework.Base" version="1.0.0-alpha-2022.7.8.2331" targetFramework="net48" />
+  <package id="HEC.MVVMFramework.Base" version="1.0.0-alpha-2022.7.20.1519" targetFramework="net48" />
   <package id="HEC.MVVMFramework.Model" version="1.0.0-alpha-2022.7.8.2331" targetFramework="net48" />
   <package id="RASMapperLibDistro6.2" version="0.0.1" targetFramework="net48" />
   <package id="System.Collections.Immutable" version="1.5.0" targetFramework="net48" />
-  <package id="USACE.HEC.FDA.Model" version="1.1.0-alpha-2022-07-19-1451" targetFramework="net48" />
```

Comment:
I'm confused how this keeps coming back, but please delete all these package references from Model, RASMapperDistro, Framework Model and Base, and statistics. Super weird. I feel like you've deleted these before.

Reply:
I will try a wooden stake and a silver bullet this time.